connector/src/main/scala/com/microsoft/kusto/spark/datasink/SparkIngestionProperties.scala

```diff
@@ -0,0 +1,65 @@
+package com.microsoft.kusto.spark.datasink
+
+import java.util
+
+import com.microsoft.azure.kusto.ingest.IngestionProperties
+import org.codehaus.jackson.annotate.JsonAutoDetect.Visibility
+import org.codehaus.jackson.annotate.JsonMethod
+import org.codehaus.jackson.map.ObjectMapper
+
+class SparkIngestionProperties {
```

Comment:
Shouldn't this inherit from IngestionProperties?

I would have a list of supported properties, and let the base class ensure that setting the value is supported instead of repeating all these functions.

Reply:
Talked F2F, ignore this.

Reply:
Talked F2F - preferred not to fail on unsupported options , and expose here only what can be used